the-collab-lab/tcl-77-smart-shopping-list
# Issue # 43 - A user should be able to be directed to a list and view a list name

## Description

Reopening up a new PR again finally closing out Issue #43 after it was opened again and a reverted PR was merged . Fixed bug issues of duplicate home page rendering and list nav bar still appearing. 

In this PR, users should be able to directly navigate to lists after clicking list buttons and list name should appear at the top of list view. This PR also highlighted a new point: list item data will only fetch new lists since our ListItemModel Type was strategically updated in a previously merged PR to include the newly defined parameter: itemQuantity. Incredible team work and insight from the whole team to uncover this new detail! 

## Related Issue

closes #43 

## Acceptance Criteria

- [x] The user is immediately redirected to the list after clicking the name of each list.
- [x] The name of the list appears at the top of each list view when a user clicks.
- [x] The user is still able to view each list from the mange-list view.

## Type of Changes

`enhancement` 

## Updates

https://github.com/user-attachments/assets/d181301d-3699-4bc6-ace7-21859f96a8b6

## Testing Steps / QA Criteria

1. Sign into application
2. Select or create a list 
3. List name will appear the top of list view
4. If list is newly created, list items will appear
5. Else, list items will not appear if it is an old list
